Sources/Plasma/PubUtilLib/plNetCommon/plNetMsgHandler.h

```diff
@@ -58,14 +56,22 @@ class plNetMsgHandler
 protected:
     plNetApp* fNetApp;
 public:
+    enum class Status
```

Comment:
FYI on Linux, building the client, I'm getting an error from this: `error: unnamed scoped enum is not allowed`

I think if this is being declared as a `enum class` it needs to "inherit" from a type like `int32_t`?

Reply:
Nevermind, this is because of X11 doing stupid stuff... 🤦🏼‍♂️ 

```
/usr/include/X11/Xlib.h:83:16: note: expanded from macro 'Status'
#define Status int
               ^
```